Micah riley

micahriley88 opened the pull request.

dustinsnoap commented on Article/Article.js:
```diff
@@ -112,3 +112,44 @@ const data = [
+data.forEach(data => {
+  articles.appendChild(createArticle(data.title, data.date, data.firstParagraph, data.secondParagraph, data.thirdParagraph));
+})
+
```
```//what does this function do?```

dustinsnoap commented on Article/Article.js:
```diff
@@ -112,3 +112,44 @@ const data = [
+  const p2 = document.createElement('p');
+  const p3 = document.createElement('p');
+  const button = document.createElement('span')
+
```
```//what is this section of the function doing?```

dustinsnoap commented on Article/Article.js:
```diff
@@ -112,3 +112,44 @@ const data = [
+  dateText.classList.add('date');
+  button.classList.add('expandButton');
+
+
```
```//What about this section?```

dustinsnoap commented on Article/Article.js:
```diff
@@ -112,3 +112,44 @@ const data = [
+  p3.textContent = thirdParagraph;
+  button.textContent = "Toggle View";
+
+
```
```//What does toggling the class "article-open" do?```

dustinsnoap commented on Menu/Menu.js:
```diff
@@ -33,3 +33,29 @@ let menuItems = [
+  let ul = document.createElement('ul');
+  header.appendChild(div);
+  div.classList.add('menu');
+
```
Good use of an array here.

micahriley88 commented:
> You code; as always, is very clean and easy to read. I would like to see more comments describing what things are doing, but that's about it. Good job.

Hey Dustin,
Thanks for going over my code for this project.
I didn't get to see this until after the Sprint, but coming from there I understand where you were coming from.
I went ahead and refactored my code (mostly the parameter and elements being passed through the component). I also added notations between sections to better emphasize what's going on.
I'm sending a new pull request now. If you get a chance to see this feel free to merge!
Thanks again!
